src/model/schema.js

```diff
@@ -465,7 +465,7 @@ export class MarkType extends SchemaItem {
 // with node type constructors and another similar object associating
 // mark names with mark type constructors.
 export class SchemaSpec {
-  // :: (?Object<NodeType>, ?Object<MarkType>)
+  // :: (?Object<SubClass<NodeType>>, ?Object<SubClass<MarkType>>)
```

Comment:
I thought so (static properties). I don't think you can use type-system classes as a good way to model classes in your domain, though. Classes in your domain would be type-system objects of a `DomainClass` type, which would just be too much useless layers. Which static properties are you using, though? Couldn't they be moved to standalone or instance properties?